lib/task/propel/propelBuildNestedSetTask.class.php

```diff
@@ -173,4 +172,4 @@ protected function reindexLft(string $classname, int $id, int $lft)
             $classname, $id, ['lft' => $lft]
         );
     }
-}
+}
```

Comment:
PHP code style checker might be failing the test because the file doesn't end with a new line character like it was before.

Reply:
ok so this one doesn't matter then? I saw some other commits with this failing so wondering when it matters and when it doesn't?